applications/GeoMechanicsApplication/custom_constitutive/small_strain_umat_3D_law.cpp

```diff
@@ -382,57 +382,51 @@ void SmallStrainUMAT3DLaw::CalculateMaterialResponseCauchy(ConstitutiveLaw::Para
     KRATOS_TRY
 
     // Get Values to compute the constitutive law:
-    const Flags& rOptions = rValues.GetOptions();
+    const Flags& r_options = rValues.GetOptions();
 
-    KRATOS_DEBUG_ERROR_IF(rOptions.IsDefined(ConstitutiveLaw::USE_ELEMENT_PROVIDED_STRAIN) &&
-                          rOptions.IsNot(ConstitutiveLaw::USE_ELEMENT_PROVIDED_STRAIN))
+    KRATOS_DEBUG_ERROR_IF(r_options.IsDefined(ConstitutiveLaw::USE_ELEMENT_PROVIDED_STRAIN) &&
+                          r_options.IsNot(ConstitutiveLaw::USE_ELEMENT_PROVIDED_STRAIN))
         << "The SmallStrainUMAT3DLaw needs an element provided strain" << std::endl;
 
     KRATOS_ERROR_IF(!rValues.IsSetStrainVector() || rValues.GetStrainVector().size() != GetStrainSize())
         << "Constitutive laws in the geomechanics application need a valid provided strain" << std::endl;
 
-    if (rOptions.Is(ConstitutiveLaw::COMPUTE_STRESS)) {
-        Vector& rStressVector = rValues.GetStressVector();
-        CalculateStress(rValues, rStressVector);
+    if (r_options.Is(ConstitutiveLaw::COMPUTE_STRESS)) {
+        Vector& r_stress_vector = rValues.GetStressVector();
+        CalculateStress(rValues, r_stress_vector);
     }
 
-    if (rOptions.Is(ConstitutiveLaw::COMPUTE_CONSTITUTIVE_TENSOR)) {
+    if (r_options.Is(ConstitutiveLaw::COMPUTE_CONSTITUTIVE_TENSOR)) {
         // Constitutive matrix (D matrix)
-        Matrix& rConstitutiveMatrix = rValues.GetConstitutiveMatrix();
-        CalculateConstitutiveMatrix(rValues, rConstitutiveMatrix);
+        Matrix& r_constitutive_matrix = rValues.GetConstitutiveMatrix();
+        CalculateConstitutiveMatrix(rValues, r_constitutive_matrix);
     }
 
     KRATOS_CATCH("")
 }
 
 void SmallStrainUMAT3DLaw::UpdateInternalDeltaStrainVector(ConstitutiveLaw::Parameters& rValues)
 {
-    const Vector& rStrainVector = rValues.GetStrainVector();
+    const Vector& r_strain_vector = rValues.GetStrainVector();
 
     for (unsigned int i = 0; i < mDeltaStrainVector.size(); ++i) {
-        mDeltaStrainVector[i] = rStrainVector(i) - mStrainVectorFinalized[i];
+        mDeltaStrainVector[i] = r_strain_vector(i) - mStrainVectorFinalized[i];
     }
 }
 
 void SmallStrainUMAT3DLaw::SetExternalStressVector(Vector& rStressVector)
 {
-    for (unsigned int i = 0; i < rStressVector.size(); ++i) {
-        rStressVector(i) = mStressVector[i];
-    }
+    std::copy_n(mStressVector.begin(), mStressVector.size(), rStressVector.begin());
```

Comment:
If we want to copy all values of `mStressVector`, I feel the more intuitive way of expressing that is:
```suggestion
    std::copy(mStressVector.cbegin(), mStressVector.cend(), rStressVector.begin());
```
Or even more directly:
```cpp
    rStressVector = mStressVector;
```

Reply:
As these are array1d, the last suggestion is not possible. Adapted the first, but with begin and end.